pyuvdata/uvdata/uvdata.py

```diff
@@ -3839,6 +3840,8 @@ def sum_vis(
         strict_uvw_antpos_check : bool
             Option to raise an error rather than a warning if the check that
             uvws match antenna positions does not pass.
+        override_params : array_like of strings
+            List of object UVParameters to omit from compatibility check.
```

Comment:
Can you clarify what happens to these parameters. I assume they take the value of the first object? Also please mention somewhere in the docstring (maybe after the first line but before the parameters)  that the `object_name` and `extra_keywords` are handled specially (and how) if they differ between the objects.